lib/joke.rb

```diff
@@ -0,0 +1,9 @@
+class Joke
+
+  attr_reader :id, :setup, :punchline
```

Comment:
but you would want one between the attr_reader line and the start of `def initialize`